### PR TITLE
Feature: Added Text-utilities Component

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added heading component.
- Added Text sizes.
- Added Font weight.
- Added Text alignment.

Live link: https://quizzical-hoover-7d122a.netlify.app/

### CODE REVIEW

File: components/text-utilities/styles.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
+:root{
+    --primary-color: #8a007e;
+    --secondary-color: #005c63;
+    --third: #040404;
```
Comment (viru2001) [inline components/text-utilities/styles.css]: you can give good name here

Comment (poojakelkar) [reply components/text-utilities/styles.css]: Thanks, Resolved this one

**Comment by devdeadviz** [comment]
Your black color tone is quite sharp, tone down it to some extent.

**Review by devdeadviz — APPROVED** [review]
Everything looks good to me! Goodwork.

**Comment by poojakelkar** [comment]
Sure, I will do that Thank you @devdeadviz

**Pull request merged**